tests/test_utils/utils.py

```diff
@@ -52,3 +60,50 @@ def mock_definitions():
         exodus_config = json.load(f)
 
     return exodus_config
+
+
+def encode_signature(data: bytes):
+    return (
+        base64.b64encode(data)
+        .replace(b"+", b"-")
+        .replace(b"=", b"_")
+        .replace(b"/", b"~")
+    )
+
+
+def build_policy(url: str, expiration: datetime):
+    datelessthan = int(datetime2timestamp(expiration))
+    condition = {"DateLessThan": {"AWS:EpochTime": datelessthan}}
+    # omit the Resource parameter, users can access all of the files
+    payload = [("Condition", condition)]
+    policy = {"Statement": [OrderedDict(payload)]}
+    return json.dumps(policy, separators=(",", ":")).encode("utf-8")
+
+
+def rsa_signer(private_key: str, policy: bytes):
+    bytes_key = bytes(private_key, "utf-8")
+    loaded_key = serialization.load_pem_private_key(
+        bytes_key, password=None, backend=default_backend()
+    )
+    return loaded_key.sign(policy, padding.PKCS1v15(), hashes.SHA1())  # type: ignore
+
+
+def generate_cookies():
```

Comment:
I've implemented signing code in exodus_lambda/functions/signer.py in #324

Reply:
That PR may enable deployment testing key(s) for the integration tests.

Reply:
@negillett 
if using the exodus_lambda/functions/signer.py and its `signer.cookies_for_policy()`.
```
        cookies_content = signer.cookies_for_policy(
            append="; Secure; Path=/content/; Max-Age=%s"
            % int(expiration.total_seconds()),
            resource="https://%s/content/*" % domain,
```
I think it requires integration tests ask for cookies for every request (`r = requests.get(url, cookies=TEST_COOKIES)`)
The AC for integration tests is one cookies for all requests, so maybe not using the new cookies endpoint in #324, WDYT?
CC: @crungehottman @rohanpm

Reply:
@dichn, 
My point is only that some of the stuff from #324 can be reused here.
I.e., you could adapt the `generate_cookies` function. Something like this should work;
```
def generate_test_cookies():
    key = os.environ.get("EXODUS_CDN_PRIVATE_KEY")
    key_id = os.environ.get("EXODUS_CDN_KEY_ID")

    if not key_id or not key:
        return {}

    signer = Signer(key, key_id)
    expiration = datetime.utcnow() + timedelta(minutes=60)
    return signer.cookies_for_policy(date_less_than=expiration)
```